app/views/casa_cases/show.html.erb

```diff
@@ -52,8 +52,12 @@
     </p>
 
     <% unless @casa_case.court_report_not_submitted? %>
+      <% if @casa_case.court_report_submitted? && @casa_case.court_report.attached? %>
```

Comment:
I was not sure whether status should be `court_report_not_submitted` or specifically `submitted` but followed the card!